backend/src/cmd/import_realm_tree.rs

```diff
@@ -220,6 +226,29 @@ fn insert_realm<'a>(
             }
         }
 
+        // Insert acl roles
+        if realm.page_admins.is_some() || realm.page_moderators.is_some() {
+            let admin_roles = match &realm.page_admins {
+                Some(roles) => roles.to_vec(),
+                _ => vec![],
+            };
+        
+            let moderator_roles = match &realm.page_moderators {
+                Some(roles) => admin_roles
+                    .clone()
+                    .into_iter()
+                    .chain(roles.to_vec().into_iter())
+                    .collect(),
+                _ => vec![],
+            };
+            let query = "update realms \
+                set admin_roles = $1, moderator_roles = $2 \
+                where id = $3 \
+            ";
+            db.execute(query, &[&admin_roles, &moderator_roles, &id]).await?;
```

Comment:
And in the case you removed the `Option` like described in my last comment, you can simplify this:

```suggestion
        if !realm.page_admins.is_empty() || !realm.page_moderators.is_empty() {
            let query = "update realms \
                set admin_roles = $1, moderator_roles = ($2 || $1) \
                where id = $3 \
            ";
            db.execute(query, &[&realm.page_admins, &realm.page_moderators, &id]).await?;
```

Main trick is to do the array concatination in SQL